lib/model/folder_sendonly.go

```diff
@@ -30,7 +31,7 @@ func newSendOnlyFolder(model *Model, cfg config.FolderConfiguration, _ versioner
 	return f
 }
 
-func (f *sendOnlyFolder) PullErrors() []FileError {
+func (f *sendOnlyFolder) PullErrors() []scanner.FileError {
```

Comment:
This feels weird, PullErrors() comming from the scanner package.
Perhaps have protocol.FileError or somewhere where it makes more sense.